### PR TITLE
Feature/vmo 5662/use base store in blocks

### PR DESCRIPTION
**Pull request by bulatgab**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -21,13 +21,13 @@
...
   "engines": {
-    "node": "^14.19.1"
+    "node": "^14"
   },
   "dependencies": {
     "@avcs/autosuggest": "^1.7.5",
     "@floip/expression-evaluator": "^2.1.0",
     "@floip/expression-parser": "^1.11.3",
-    "@floip/flow-runner": "^1.0.0-rc4",
+    "@floip/flow-runner": "https://github.com/FLOIP/flow-runner.git#3a3020c3b14a27d14b62003c3fc54270cf287ad9",
```
Comment (bulatgab) [inline package.json:30]: see https://github.com/FLOIP/flow-runner/tree/feature/VMO-5662/use-base-store-in-blocks

Comment (safydy-r) [reply package.json:30]: @bulatgab 
- [ ] Once we bumped flow-runner version & we published it, let's re-use the semantic version here.

**Review by safydy-r — APPROVED** [review]
I have no concern about this approach. But I'd like to involve @jcbashdown here and get his approval as he was the one who were creating the ticket/tech AC.

- [ ] get Jacob's approval on this approach
- [ ] Fix TS error after `yarn serve`, I noticed 01 error `Module '"@floip/flow-runner"' has no exported member 'IReadBlock'.` 
- [ ] Make sure all stories in storybook are working (Should have no console error in browser). I haven't tested this.

File: src/store/flow/block-types/Core_OutputBlockStore.ts [hunk 2]
```diff
@@ -1,61 +1,39 @@
...
-        value: null,
-      },
-      exits,
-      tags: [],
-      vendor_metadata: {},
-    })
+  async createWith({dispatch}, {props}: { props: { uuid: string } & Partial<IOutputBlock> }) {
+    props.type = BLOCK_TYPE
+    props.config = {
+      value: '@(null)',
```
Comment (safydy-r) [inline src/store/flow/block-types/Core_OutputBlockStore.ts:21]: @bulatgab , I think we should keep `null` like in master https://github.com/FLOIP/flow-builder/blob/master/src/store/flow/block-types/Core_OutputBlockStore.ts#L36, that will provide us the right validation message at the beginning. Your thoughts ?
Screen Shot 2022-05-25 at 9 07 44 AM

Comment (bulatgab) [reply src/store/flow/block-types/Core_OutputBlockStore.ts]: Upd: changed to `value: null` as suggested

**Review by jcbashdown — APPROVED** [review]

**Review by yeboaasare — APPROVED** [review]

**Comment by bulatgab** [comment]
Travis CI build is failing with `Module '"@floip/flow-runner"' has no exported member 'IReadBlock'`.

**Merging anyway**

Because probably, yarn is failing to fetch the latest version of flow-runner: `1.0.0-rc4.4`.
(The same happens on local machines and we have to remove flow-runner from package.json, run yarn install, add flow-runner back, and rerun yarn install - to resolve this issue).

**Comment by bulatgab** [comment]
Upd: `yarn.lock` was not up to date. Fixed, rebuilt, pushed to `master` branch directly.

**Pull request merged**